LaLauncher/src/main/java/com/github/postapczuk/lalauncher/AllAppsActivity.java

```diff
@@ -42,4 +60,11 @@ public void onSwipeBottom() {
             }
         });
     }
+
+    // Set the left padding at the item level vs the listview level
+    private void applyItemPadding(TextView item){
```

Comment:
duplicated private method (here and in MainActivity). Move it to the AppsActivity with proper access (remove `private` word)